doc/level1.rst

```diff
@@ -637,29 +639,45 @@ Key Gossip
 ++++++++++
 
 It is a common use case to send an encrypted mail to a group of
-recipients. To ensure that these recipients can encrypt messages when
-replying to that same group, the keys of all recipients can be
-included in the encrypted payload. This does not include BCC
+recipients. To help these recipients to send encrypted replies
+to the same group, the keys of all recipients can be
+included in the encrypted payload through adding ``Autocrypt-Gossip``
+headers for each recipient . This does not include BCC
 recipients, which by definition must not be revealed to other
 recipients.
 
-The ``Autocrypt-Gossip`` header has the format as the ``Autocrypt``
+The ``Autocrypt-Gossip`` header has the same format as the ``Autocrypt``
 header (see `autocryptheaderformat`_). Its ``addr`` attribute
 indicates the recipient address this header is valid for as usual, but
 may relate to any recipient in the ``To`` or ``Cc`` header.
-See example in :ref:`autocrypt-gossip-example`
+See example in :ref:`autocrypt-gossip-example`.
+
+.. _`gossip-injection`:
 
 Key Gossip Injection in Outbound Mail
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-An Autocrypt MUA MAY include ``Autocrypt-Gossip`` headers in messages
+A MUA MAY include ``Autocrypt-Gossip`` headers in messages
```

Comment:
should be "An MUA" -- but why remove Autocrypt here?  we're talking about Autocrypt-enabled MUAs, right?